app/models/bus.rb

```diff
@@ -0,0 +1,9 @@
+class Bus < ApplicationRecord
+  validates :name, format: { with: /\A\S*\z/, message: 'no whitespace is allowed' }
+  validates :name, uniqueness: { case_sensitive: false }
+  validates :name, :typed, :brand, :capacity, presence: true
+
+  validates :capacity, numericality: { greater_than: 0, less_than_or_equal_to: 45 }
+
+  enum typed: { ac: 'ac', non_ac: 'non_ac' }
```

Comment:
Can we use same format for all `enum` s in a project? 🤔

Reply:
format changed.